### ISSUE TITLE
Acapela voices don't laugh or cry anymore

### ISSUE THREAD

**Issue by mikebayus** [nvaccess/nvda#8750]
Acapela voices are supposed to be able to laugh, cry etc.  On the NVDA addon blog, there is a fix for this to make this happen in NVDA.  Now the fix does not work any more. http://www.acapela-nvda.com/blog/2017/01/12/is-your-acapela-voice-laughing-or-crying-with-nvda/

**Comment by Mohamed00** [comment]
Also, the file mentioned in the blog post gets reverted on every update since it's in program files. You'll need to edit it every time you update, but there is also an addon that allows you to toggle speech dictionary processing with a keystroke. I'd be reluctant to link to it here, though, since it's not official.

**Comment by mikebayus** [comment]
Yes I know that.

I eddit the file mentioned in the blog every time I update.

It worked every time until I upgraded my Computer.

When I got my new computer and did a clean install, and edited builtin
dot dick, the change did not work.

I contacted acapela and told them this, and they say, "oh yes. you are
right, it doesn't work with NVDA 2018.1 and later.  But it is not our
fault.".

NVDA broke it, now FIX IT.

**Comment by mikebayus** [comment]
For Joseph Lee.

Hi Joseph,

First of all, thank you for your response.

I believe it to be most fair and reasonable.

I will forward your remarks to acapela, because I told them about the
issue first, and as I said earlier in this thread they said I am
right, that the modification no longer works.

I assumed that they meant that they would do what ever they could to
{"fix' the issue.

They have suggested to me that it was something that the devs of NVDA
did to break that function of the Addon.

And, as they did nothing to the program on their end, it was NVDA's fault.

But Joe, to admonish me for writing fix it in all caps only frustrates
me further.

You are right.

To express it that way does amount to a forceful command, or a demand.

That is what I entended.

Not to put all parties on a bad light, as I am the original poster,
but it was the only way I could think of to express my frustration
about the whole issue.

acapela touts their voices as being able to express themselves and be
able to laugh, cry, cough, etc.

It really bothers me that you can do that in Infovox, or any AAC
device that acapela is used for, but not in NVDA.

acapela developed the addon for NVDA, and, they support NVDA, so I
would think that NVDA would cooperate with them to give the addon it's
full functionality.

Some of this is subjective I know, because Ella is the only voice I
listen too now.

Because I like Ella so much, it is indeed, hard for me to be objective about it.

So yes, I am frustrated.

This functionality never worked out of the box, and should have been
addressed at the time of beta testing back in 2014.

Am I the only one who actually uses the addon?

Am I crazy for bonding with Ella the way I have?

 You say:

"Personally, I would close this issue because of that kind of
communication, but I won't because we would prefer to see a fix for
you and others affected by it, thus proving that we do care."

Glad to hear it.

But you know, if you come to think of it, because this is an issue
tracker that is visible around the world,  writing "fix it" in all
caps was a way to let every one know of my frustration.

Now you know.

**Comment by Brian1Gaff** [comment]
Grin, I had no idea this was even possible in fact.
 However could it be that they are not aware of the changes needed to get 
nvda ready for python3? If they are not going to adapt software slightly to 
run in the same way with new versions of screenreaders then that seems short 
sighted. It cannot be the first time nor the only company to subtly change 
the way they handle add ons and drivers, heck Microsoft are always at it!


As for bonding with a voice. Its a personal thing which is why I never moan 
except light heartedly about those I cannot stand like the old one that 
starts with E!
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by mikebayus** [comment]
Boy do I feel like a fool.  It turns out that the problem was that my NVDA was set to "User Default", instead of "English."  A message from Acapela gave me the idea to look at the Language setting, and I discovered that my Language was set to "User Default".  I change the setting to English, saved configuration, and now Ella is back, and fully functional again.  I would have never thought of that.  The developers of the Addon didn't either.  Problem solved, but boy do I feel foolish.

**Comment by josephsl** [comment]
Hi, I see. I think the add-ons community should document this issue somewhere. Thanks.

**Comment by mikebayus** [comment]
I agree.

That's a good idea.

I just finished telling Acapela, and maybe they will say something on
their blog.

I do hope so.

You, I think, would know better how to document this for the NVDA
addon community.

We still need to edit the builtin "dot" dic file however.

I think that my NVDA menu setting was set to English on my old Del,
and when I installed NVDA on my new HP, the language setting defaulted
to "User Default".

I am sorry I was so much trouble, but it really bothered me.

Thank you all for your patients.

Keep on keeping on, and all the very best.
Michael Bayus
USA

**Comment by Adriani90** [comment]
@mikebayus thanks for reporting. I close this one as works for me, it seems your problem is solved. If you have suggestions for editing the .dic file, please create a new feature request for it anf fill the template. Thanks.
But I also think that such issues are also synthesizer specific and not really NVDA issues. If you encounter such issues with multiple synthesizers, then it could be more related to NVDA.

**Comment by Adriani90** [comment]
Note that if something changes in NVDA, addon developers must come with an update accordingly. It is not necessarely a fault in NVDA if an addon developer does not wnat to invest time in understanding the changes and comming with a solution.
